packages/picasso.js/src/core/chart-components/axis/axis-arc-label-node.js

```diff
@@ -86,5 +97,6 @@ export default function buildArcLabels(tick, buildOpts) {
   appendStyle(struct, buildOpts);
   appendBounds(struct, buildOpts);
   appendCollider(struct, tickPos);
+  struct.maxWidth = calculateMaxWidth(buildOpts, side, innerPos);
```

Comment:
I think it's cleaner to have this on line 80 - `maxWidth: calculateMaxWidth(buildOpts, side, innerPos);`